src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../types';
+import { findPersonByName } from '../utils/findPersonByName';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
```

Comment:
It's a good practice to provide comments for your interfaces and types. This will help others (and you in the future) to understand what this interface is for and what each property means.